LAYOUT.md

```diff
@@ -74,6 +73,11 @@ HTML that are used in several page layouts.  These directories have
 underscores at the start of their names to be consistent with Jekyll's
 naming conventions, but the files they contain are for Pandoc.
 
+Most authors should not need to modify any of the support files themselves.
+The only exception is the `_includes/footer.html` file linking the "Source"
+button to [https://github.com/swcarpentry/lesson-template][swc-lesson-template].
```

Comment:
No need to show folks the whole URL, just use the project name:

    … linking the "Source" button to [lesson-template][swc-lesson-template].